src/components/navbar/NavBar.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
```

Comment:
If you rename this file to `index.tsx`, you can import it in App.tsx via: 
`import NavBar from './components/navbar`
Basically if you import a directory using that syntax, it pulls the index.

Reply:
Oooh neat trick. Thanks!